styles.css

```diff
@@ -1,3 +1,7 @@
+* {
+  outline: auto;
```

Comment:
- [ ] **[OPTIONAL]** Please, 🙏🏽 we could avoid using `outline: auto` to delete the border around elements.

- With outline

![With-outline](https://user-images.githubusercontent.com/58642949/186296877-332948a9-18b4-490d-a6b8-aef6a9fe40c2.png)

- Without outline

![Without-outline](https://user-images.githubusercontent.com/58642949/186296914-d037b4fb-2457-4aaa-863a-8438f1347096.png)